docs/sphinx/platforms.rst

```diff
@@ -0,0 +1,64 @@
+Supported platforms
+===================
+
+The OME C++ software components are written using portable,
+cross-platform C++.  The components and their dependencies are tested
+on a daily basis and are proven to work on a set of core platforms
+(they build without error and pass their unit and integration tests).
+Builds are provided for each release for these platforms.  The support
+status for these is as follows:
+
++---------------+--------------+--------------+--------------+-------------+
+|               |           Version           |         Build type         |
++---------------+--------------+--------------+--------------+-------------+
+| System        | Recommended  | Minimum      | Platform     | Super-Build |
++===============+==============+==============+==============+=============+
+| CentOS        | 7            | 6            | Unsupported* | Supported   |
++---------------+--------------+--------------+--------------+-------------+
+| Debian        | 8            | 7            | Unsupported  | Unsupported |
++---------------+--------------+--------------+--------------+-------------+
+| FreeBSD       | 11           | 10           | Supported†   | Supported   |
```

Comment:
No, this one verified by hand (I've started a test build to confirm, and we can add it back to the matrix if it's OK).  This is the same for Ubuntu 16.04; the building here is local in the absence of CI support.